ts-redis/index.ts

```diff
@@ -12,24 +12,32 @@ import { createClient } from 'redis';
 *   id = "redis"
 * }
 */
-const client = createClient({});
+
+const setupRedisClient = async () => {
+  const client = createClient({socket: {
```

Comment:
The persist annotation should go on this line now, right?

Reply:
yeah it should, but it actually still works because it picks up that the query matches a child node of the node thats annotated. I will switch it though for clarity, but kind of cool that we can traverse down and match